test/test_local.py

```diff
@@ -30,10 +30,33 @@ def tearDownClass(cls):
         cls.network2.disconnect()
 
     def test_expedited_upload(self):
-        self.local_node.sdo["Producer heartbeat time"].raw = 0x99
-        vendor_id = self.remote_node.sdo["Producer heartbeat time"].raw
+        self.local_node.sdo[0x1400][1].raw = 0x99
+        vendor_id = self.remote_node.sdo[0x1400][1].raw
         self.assertEqual(vendor_id, 0x99)
 
+    def test_block_upload_switch_to_expedite_upload(self):
+        with self.assertRaises(canopen.SdoCommunicationError) as context:
+            self.remote_node.sdo[0x1008].open('r', block_transfer=True)
+        # We get this since the sdo client don't support the switch
+        # from block upload to expedite upload
+        self.assertTrue("Unexpected response 0x41" in context.exception)
```

Comment:
Convert `context.exception` to `str` first (see [Travis](https://travis-ci.org/christiansandberg/canopen/jobs/332983801)).

Reply:
Fixed